From: Etienne Millon <me@emillon.org>
Date: Mon, 13 Aug 2012 19:52:50 +0200
Subject: Disable LyrDB lyrics provider

LyrDB.com makes gmpc segfault, and there are other lyrics providers so disabling
it is a net profit.

Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=650359
---
 src/Tools/plugin-man.c |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/Tools/plugin-man.c b/src/Tools/plugin-man.c
index 621bb00..197bcbd 100644
--- a/src/Tools/plugin-man.c
+++ b/src/Tools/plugin-man.c
@@ -116,9 +116,6 @@ void plugin_manager_load_internal_plugins(void)
             gmpc_provider_render_cover_new(),
             0, NULL);
     plugin_add_new((GmpcPluginBase *) 
-            gmpc_provider_lyr_db_new(),
-            0, NULL);
-    plugin_add_new((GmpcPluginBase *) 
             gmpc_provider_chart_lyrics_new(),
             0, NULL);
 }
